app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -65,6 +65,9 @@ echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 <% end -%>
 
 <%= snippet 'preseed_networking_setup' -%>
+
+<%= snippet_if_exists(template_name + " custom late snippet") %>
+
```

Comment:
Please don't add any new lines, use `-%>` for the end tag.

Also the suffix name, probably just " custom" if you don't mind. This finish script is just a shell script, no need for more complicated names I guess:

```
app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb:<%= snippet_if_exists(template_name + " custom menu") %>
app/views/unattended/provisioning_templates/provision/kickstart_default.erb:<%= snippet_if_exists(template_name + " custom repositories") %>
app/views/unattended/provisioning_templates/provision/kickstart_default.erb:<%= snippet_if_exists(template_name + " custom packages") %>
app/views/unattended/provisioning_templates/provision/kickstart_default.erb:<%= snippet_if_exists(template_name + " custom pre") %>
app/views/unattended/provisioning_templates/provision/kickstart_default.erb:<%= snippet_if_exists(template_name + " custom postnochroot") -%>
app/views/unattended/provisioning_templates/provision/kickstart_default.erb:<%= snippet_if_exists(template_name + " custom post") %>
app/views/unattended/provisioning_templates/provision/coreos_provision.erb:  #{indent(2) { snippet_if_exists('CoreOS provision custom passwd') }}
app/views/unattended/provisioning_templates/provision/coreos_provision.erb:  #{indent(2) { snippet_if_exists('CoreOS provision custom systemd') }}
app/views/unattended/provisioning_templates/provision/coreos_provision.erb:  #{indent(2) { snippet_if_exists('CoreOS provision custom storage') }}
app/views/unattended/provisioning_templates/provision/coreos_provision.erb:#{snippet_if_exists('CoreOS provision custom')}
app/views/unattended/provisioning_templates/provision/atomic_kickstart_default.erb:<%= snippet_if_exists(template_name + " custom pre") %>
app/views/unattended/provisioning_templates/provision/atomic_kickstart_default.erb:<%= snippet_if_exists(template_name + " custom post") %>
app/views/unattended/provisioning_templates/finish/kickstart_default_finish.erb:<%= snippet_if_exists(template_name + " custom pre") -%>
app/views/unattended/provisioning_templates/finish/kickstart_default_finish.erb:<%= snippet_if_exists(template_name + " custom post") -%>
app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb:<%= snippet_if_exists(template_name + " custom snippet") %>
app/views/unattended/provisioning_templates/PXEGrub2/preseed_default_pxegrub2.erb:<%= snippet_if_exists(template_name + " custom menu") %>
app/views/unattended/provisioning_templates/PXEGrub2/kickstart_default_pxegrub2.erb:<%= snippet_if_exists(template_name + " custom menu") %>
app/views/unattended/provisioning_templates/PXELinux/preseed_default_pxelinux.erb:<%= snippet_if_exists(template_name + " custom menu") %>
app/views/unattended/provisioning_templates/PXELinux/kickstart_default_pxelinux.erb:<%= snippet_if_exists(template_name + " custom menu") %>
```

Reply:
Hi @lzap !  Before I fix the spacing -- are you simply referring to removing the empty lines, or should the snippit call also have -%> at the end?  I worry that the latter would potentially cause problems if the called snippit didn't end with an empty line at the end.

As far as the naming -- I only went with that because of the OTHER custom snippit entry in there.  Further up the preseed finish template there is " custom snippit"  I wasn't sure what else to call a .. later version of that .. so I just added late to it.  Any thoughts on what to call it that would make sense based off the other one existing?

I'm referring to this of course:
`app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb:<%= snippet_if_exists(template_name + " custom snippet") %>`

What I'm adding is another, later, hook here.  Another option I could do is to change mine to "custom finish" and leave the old custom snippit in there.

Reply:
I'm happy to make those changes btw -- I just want to confirm that I'm not missing something or a potential problem.  =)  Ignore my suggestion of 'custom finish' though because that's. . well dumb once I reread over it.  I guess the core TL;DR questions I have for you are:

1. Are you sure the -%> won't cause problems if the custom template doesn't end in a newline?
2. Should we just ignore the "custom snippit" one that already exists and call this one "custom" and let them both exist for now?